thirdeye-ui/src/app/locale/languages/en-us.json

```diff
@@ -630,6 +632,7 @@
         "weeks-lowercase": "weeks",
         "what-went-wrong-and-where": "What went wrong and where?",
         "whole-alert": "Whole alert",
+        "workspace-config": "workspace config",
```

Comment:
this is user facing right ? 
should we use Settings instead also here ?

Reply:
Yes, let me change it to settings